src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,10 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
+    } 
+    public static void trim(String []str){
```

Comment:
And I don't think this will work. Let's make it return a String array as result.